docs/src/main/asciidoc/security-testing.adoc

```diff
@@ -121,6 +121,38 @@ If it becomes necessary to test security features using both `@TestSecurity` and
 mechanism when none is defined), then Basic Auth needs to be enabled explicitly,
 for example by setting `quarkus.http.auth.basic=true` or `%test.quarkus.http.auth.basic=true`.
 
+=== Path-based authentication
+
+`@TestSecurity` can also be used when xref:security-authentication-mechanisms.adoc#combining-authentication-mechanisms[authentication mechanisms must be combined].
+Example below shows how to select authentication mechanism when path-based authentication is enabled by HTTP Security
```

Comment:
I propose to promote the new way of using annotations, so, start with one example where 2 annotations are used only, no policies. And the 2nd example is the current one (but please show the policy requiring form as well)

Reply:
IMO done, thanks.